### DISCUSSION TITLE
Stock Test Results - Allowing Assemblies to Inherit Test Results

### DISCUSSION THREAD

**Discussion by vk2tds** [inventree/InvenTree#7286] [Ideas]
I have a scenario where I am tracking serial numbers on devices. However, I also track the MAC address and firmware version. I am tracking the MAC and firmware as test results on the Raw PCB. I then add connectors and antistatic bag, creating an OEM PCB. I then add retail packaging to create a Retail PCB 

`Retail PCB -> OEM PCB -> Raw PCB`

When I click on the Retail PCB (or OEM PCB) and go to Test Results, it would be great to see the test results (with numbers) from the Raw PCB. I guess I could build a plug-in that actually copied the results up the chain, but it would be great to see this as core functionality.

**Comment by SchrodingersGat** [comment]
@vk2tds this functionality is already in place, but when I check now it appears to be not working correctly...

Personally I have been using such a feature for a number of years - a test report for an assembled electronic board includes results for multiple installed items (such as firmware version, etc, etc).

~~I'll look into why the test table is not showing the installed results correctly. That's a bug.~~

*Edit: There was no bug, it is working as intended (see image below).*

In the image below you can see that with "installed items" included in the test result table, you get the following:

![image](https://github.com/inventree/InvenTree/assets/10080325/dafe0fd4-8412-4c65-9008-55244ab5e474)

And with "installed items" filter disabled:

![image](https://github.com/inventree/InvenTree/assets/10080325/d5f93eff-ff6c-4d23-a4cd-d2dd9b4b71a8)
